### ISSUE TITLE
No way to pass title metadata to MPC HC

### ISSUE THREAD

**Issue by johnzach** [webtorrent/webtorrent-desktop#1349]
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
0.19.0

**What operating system and version?**
Windows 10 

**What did you do?**
Open external player

**What did you expect to happen?**
have the name of the torrented video file as a title

**What actually happened?**
Had '1' as a title or "localhost" which prevents me from autosearching subtitles

Is there a way to pass actual video file title to the player?

**Comment by feross** [comment]
If it's showing `1` then I think that it's fixable. In the WebTorrent server we expose two types of URLs:

```
https://localhost:port/<file-index>
https://localhost:port/<file-index>/<file-name>
```

Which are equivalent, but the latter can be useful when a file name is being inferred by the URL. We just need to switch to the latter URL format to improve the situation it seems.

This should be a relatively easy fix.

**Comment by hicom150** [comment]
Should be fixed in #1648 😉

**Comment by feross** [comment]
Fixed by #1648
